test/lint.js

```diff
@@ -4,7 +4,7 @@ const lint = require('mocha-eslint');
 // Note: a seperate Mocha test will be run for each path and each file which
 // matches a glob pattern
 const paths = [
-  'server',
+  'src',
```

Comment:
🤔

Reply:
Whoops I accidentally move the .git folder into server, so client never got committed. All fixed now